api/test/auth_test.py

```diff
@@ -52,7 +53,11 @@ def test_auth_token_exchange(client, test_auth_tokens_registration):
 
 
 @pytest.mark.django_db
-def test_auth_email_verification(client, test_auth_token_exchange):
+@pytest.mark.parametrize(
+    "rate_limit_model",
+    [x[0] for x in ThrottledApplication.RATE_LIMIT_MODELS],
+)
```

Comment:
Not critical, but this could be set as a variable and used for each test that requires it. E.g.:

```python
rate_limit_model_params = pytest.mark.parametrize(
    "rate_limit_model",
    [x[0] for x in ThrottledApplication.RATE_LIMIT_MODELS],
)


@pytest.mark.django_db
@rate_limit_model_params
def test_auth_email_verification
```

Reply:
Learned something new!